.circleci/config.yml

```diff
@@ -12,7 +12,8 @@ jobs:
             - checkout
             - run: sudo pip install .[sklearn,tf-cpu,torch,testing]
             - run: sudo pip install codecov pytest-cov
-            - run: python -m pytest -n 8 --dist=loadfile -s -v ./tests/ --cov
+            - run: python -m pytest -n 8 --dist=loadfile -s -v ./tests/test_modeling_tf* -k model_equivalence --cov
+            - run: python -m pytest -sv ./tests/ -k batch_encode_plus_tensors --cov
```

Comment:
this is faster without parralellism

Reply:
I find this whitelist tricky to maintain into the future. Maybe we can simplify it with multiple `-k` filters (in a single command)

Reply:
pytest doesn't support multiple -k filters yet https://stackoverflow.com/questions/13295627/can-py-test-support-multiple-k-options